src/main/java/me/A5H73Y/parkour/utilities/Utils.java

```diff
@@ -42,6 +21,11 @@
 import org.bukkit.inventory.meta.ItemMeta;
 import org.bukkit.material.Stairs;
 
+import java.io.*;
+import java.text.Format;
+import java.text.SimpleDateFormat;
+import java.util.*;
+
```

Comment:
Please use checkstyles, this is a bad practice to import wildcards for various reasons.